agent/fw_drupal8.c

```diff
@@ -529,6 +530,21 @@ NR_PHP_WRAPPER(nr_drupal8_name_the_wt_via_symfony) {
 }
 NR_PHP_WRAPPER_END
 
+void nr_drupal_version(TSRMLS_D) {
+  char* string = "Drupal::VERSION;";
+  zval retval;
+  int result
+      = zend_eval_string(string, &retval, "Retrieve Drupal Version" TSRMLS_CC);
+  if (result == SUCCESS) {
+    if (Z_TYPE(retval) == IS_STRING) {
+      char* version = Z_STRVAL(retval);
+      // Add php package to transaction
+      nr_txn_add_php_package(NRPRG(txn), "drupal/core", version);
+      zval_dtor(&retval);
+    }
+  }
```

Comment:
Should this function still register drupal w/o a version if this function is called?

Reply:
Yes, thanks for catching that. fixed: [3eaa90f](https://github.com/newrelic/newrelic-php-agent/pull/775/commits/3eaa90f72f9976ab895574749ae180d1acad0c0a)

Reply:
Does this work for Drupal 7?

Reply:
No, this only works for Drupal 8 and above